core/templates/components/summary-tile/topic-summary-tile.directive.html

```diff
@@ -60,6 +62,11 @@
     z-index: 100;
   }
 
+  .fx-row {
+    display: flex;
+    justify-content: center;
+  }
```

Comment:
It is not great to overrid properties of a oppia.css class. Maybe we can use `fx-main-center` class directly in the element we want to be centered?

Reply:
Done.